tests/coordinates/test_path_finder.py

```diff
@@ -0,0 +1,390 @@
+import time
+
+import astropy.constants as const
+import astropy.units as u
+from astropy.coordinates import AltAz, EarthLocation, FK5
+from astropy.time import Time
+import pytest
+
+from neclib.coordinates import CoordCalculator, PathFinder
+
+
+class TestPathFinder:
+
+    location = EarthLocation("138.472153deg", "35.940874deg", "1386m")
+    pressure = 850 * u.hPa
+    temperature = 290 * u.K
+    relative_humidity = 0.5
+    obswl = const.c / (230 * u.GHz)
+    obsfreq = 230 * u.GHz
+    # config.antenna_command_frequency = 50
+    # config.antenna_command_offset_sec = 3
+
+    def expected_value(self, pointing_param_path, lon, lat, frame, unit, obstime):
+        calculator = CoordCalculator(
+            location=self.location,
+            pointing_param_path=pointing_param_path,
+            pressure=self.pressure,
+            temperature=self.temperature,
+            relative_humidity=self.relative_humidity,
+            obsfreq=self.obsfreq,
+        )
+        az, el, _ = calculator.get_altaz(
+            lon=lon,
+            lat=lat,
+            frame=frame,
+            unit=unit,
+            obstime=obstime,
+        )
+        return az.value * az.unit, el.value * el.unit, obstime
+
+    def test_01_sample(self, data_dir):
+        pointing_param_path = data_dir / "sample_pointing_param.toml"
+        finder = PathFinder(
+            location=self.location,
+            pointing_param_path=pointing_param_path,
+            pressure=self.pressure,
+            temperature=self.temperature,
+            relative_humidity=self.relative_humidity,
+            obswl=self.obswl,
+        )
+        now = time.time()
+        az, el, t = finder.linear(
+            start=(0, 0),
+            end=(0.05, 0),
+            frame="altaz",
+            speed=0.5,
+            unit=u.deg,
+        )
+        expected_az, expected_el, expected_t = self.expected_value(
+            pointing_param_path=pointing_param_path,
+            lon=[0, 0.01, 0.02, 0.03, 0.04, 0.05],
+            lat=[0, 0, 0, 0, 0, 0],
+            frame="altaz",
+            unit=u.deg,
+            obstime=[
+                now + 3,
+                now + 3.02,
+                now + 3.04,
+                now + 3.06,
+                now + 3.08,
+                now + 3.1,
+            ],
+        )
+        assert az.value == pytest.approx(expected_az.value)
+        assert az.unit == expected_az.unit
+        assert el.value == pytest.approx(expected_el.value)
+        assert el.unit == expected_el.unit
+        assert t == pytest.approx(expected_t)
+
+    def test_02_sample_another_type(self, data_dir):
```

Comment:
わかりやすい関数名に変更してほしいです。
```suggestion
    def test_quantity_input(self, data_dir):
```

Reply:
レビューありがとうございます。
ご指摘の通りテストコードの関数名を少し変更しました。